### PR TITLE
systemd: Add a service API (and use it).

### PR DESCRIPTION
**Pull request by mvollmer**
And a test page for it.

API docs and tests are missing.

### CODE REVIEW

**Comment by mvollmer** [comment]
The idea is to use this for the pmlogger button on the host page and for controlling multipathd in storage.

File: pkg/systemd/service.js [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+     *
+     * A boolean that tells whether the service is known or not.  A
+     * proxy with 'exists == false' will have 'state == undefined' and
+     * 'enabled == undefined'.
+     *
+     * - proxy.state
+     *
+     * Either 'undefined' when the state can't be retrieved, or a
+     * string that has one of the values "starting", "running",
+     * "stopping", "stopped", or "failed".
```
Comment (dperpeet) [inline pkg/systemd/service.js]: or null, when receiving an unexpected value from the system

Comment (mvollmer) [inline pkg/systemd/service.js]: Good catch!  The docs are right, the code is wrong.

Comment (mvollmer) [inline pkg/systemd/service.js]: Fixed.

**Comment by stefwalter** [comment]
Test failure is being handled in #2627

**Pull request closed without merging**